utils/webScraper.js

```diff
@@ -1,89 +1,85 @@
-/* eslint-disable no-console */
 /* eslint-disable no-plusplus */
 const cheerio = require('cheerio');
 const axios = require("axios");
-
 // For Event link reference, add scraped ID to visit the event page
 // const linkFormat = `https://www.eventbrite.com/e/neurips-meetup-port-harcourt-tickets-${'link'}?aff=ebdssbdestsearch`;
 // variable samples
 const science = 'science-and-tech';
 // const travel = 'travel-and-outdoor';
 const locationSample = 'port-harcourt';
 // variable samples
+const fetchData = async () => {
+  const result = await axios({
+    method: "get",
+    url: `https://www.eventbrite.com/d/nigeria--${locationSample}/${science}--events/`,
```

Comment:
This `URL` should be more dynamic and location should not be hardcoded

Reply:
Yes. This will be refactored. 
We have created a card for making the scraping dynamic(https://trello.com/c/FOeY3RSV/48-make-scaping-more-dynamic). That will be our focus for tomorrow.

This is static for now.